### DISCUSSION TITLE
Problems with 7.21.0

### DISCUSSION THREAD

**Discussion by adrox37** [babel/babel#15441] [Releases] [answered]
I am unable to resolve 7.21.0 and seeing that there are 0 downloads in npmjs:

notarget No matching version found for @babel/parser@^7.21.0
<img width="873" alt="image" src="https://user-images.githubusercontent.com/479607/220163680-63c76292-579b-4e7c-b672-afae4d1a90fb.png">

notarget No matching version found for @babel/types@^7.21.0
<img width="812" alt="image" src="https://user-images.githubusercontent.com/479607/220163610-dea4e11a-2aec-4546-80ff-04298596400c.png">

**Comment by JLHwung** [comment, marked as answer]
Hi @adrox37!
This seems to be an issue caused by a package that you are not able to download. You can verify that the package is available using the npm website: for example, https://www.npmjs.com/package/@babel/types?activeTab=versions is the URL for `@babel/types` (replace `@babel/types` in the URL with the package you want to check). If you see zero downloads don't worry: npm can take up to a few days to display the downloads count, but this does not mean that the package is not available.
Here are some things you can try:
1. Please check if there is a stale proxy or cache between you and npm (this is the most likely cause and it is common in corporate networks).
2. Visit https://status.npmjs.org/ to see if the current npm is working properly.
3. See https://gist.github.com/hzoo/df94af2d2ec48696e683b772c775aa67
